climpred/preprocessing/mpi.py

```diff
@@ -0,0 +1,118 @@
+import glob as _glob
+import os as _os
+
+import numpy as np
+import xarray as xr
+
+
+def get_path(
+    dir_base_experiment='/work/bm1124/m300086/CMIP6/experiments',
+    member=1,
+    init=1960,
+    model='hamocc',
+    output_stream='monitoring_ym',
+    timestr='*1231',
+    ending='nc',
+):
+    """Get the path of a file of for MPI-ESM standard output file names and directory.
```

Comment:
Typo here. Should be:

"Get the path of a file for MPI-ESM standard output file names and directory."